distance_validation_test.py

```diff
@@ -26,3 +30,31 @@ def test_float(self):
         self.assertAlmostEqual(l2_distance(user_two_attributes, user_one_attributes), 1.280624847)
 
 #insert classes for other distance functions we create and need to test
+#Test Cosine Distance Class created by Kayla Patterson 07/14/2020
+class TestCosineDistanceFunction(unittest.TestCase): 
+    def test_zero_distance(self):
+        #Ensure that two users that are the same is equal to 1 
+        #Cosine Distance is different from Euclidean since when users are the same the
+        #distance will be the max distance which is 1
```

Comment:
it isn't distance then right? it is similarity? I assume 1 means that they are the same. Maybe if you want it to be distance you should do 1-similarity? Otherwise, looks good

Reply:
Yes, you are right. I meant to say the similarity not distance. I will change it to 1-similarity.